example/storybook/stories/Notifications/index.tsx

```diff
@@ -66,9 +56,13 @@ const styles = StyleSheet.create({
 });
 
 export const NotificationsScreen = () => {
-  const [events, setEvents] = useState<Event[]>([]);
-  const { httpClient } = useHttpClient();
-  const { account } = useActiveAccount();
+  const pushNotificationsContext = useContext(PushNotificationsContext);
+
+  if (!pushNotificationsContext) {
+    return <Text>Context is undefined</Text>;
```

Comment:
Couple of things here - probably dont want to explicitly check that the context is defined here (and even so probably want to do something else in the case where it is undefined)

Going to lean into yalls expertise for this piece as well